### DISCUSSION TITLE
When I try to use HTTP API it just says "error": "Unauthorized"

### DISCUSSION THREAD

**Discussion by thetejasmagade** [deta/cloud-docs#501] [Help]
When, I try to use `HTTP API` it just says `"error": "Unauthorized"`, 
but If I do it with SDK it's working fine.

![Screenshot 2022-09-25 135103](https://user-images.githubusercontent.com/53045273/192134663-b36cc3f2-6d7e-4198-9d36-4b256b3efe52.jpg)

**Comment by crabart** [comment]
You need to add X-Api-Key to http headers. See [Auth section](https://docs.deta.sh/docs/drive/http#auth).

e.g.
Using curl, `curl -O -H "X-Api-Key: project-key" https://drive.deta.sh/v1/project-id/drive-name/files/download?name=demo-deta-img.jpg`.

Using Google Chrome, it needs to use add-ons, such as https://chrome.google.com/webstore/detail/modheader/idgpnmonknjnojddfkpgkljpfnnfcklj
